### PR TITLE
Fixes to the component chapter

### PR DESCRIPTION
**Pull request by torkleyy**

### CODE REVIEW

File: book/src/concepts/component.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Here's a comparison of the most used ones:
 * `DenseVecStorage`: Elements are stored in a contiguous array. No empty space is left between `Component`s, allowing a lowered memory usage.
   This is less performant than `VecStorage`.
 * `VecStorage`: Elements are stored into a contiguous array. Uses more memory than `DenseVecStorage` but is more performant when you use a `Component` type a lot.
-  Only use this when you know that a LOT of your entities are going to have that `Component` type. (More than 90%, for example).
+  Only use this when you know that many of your entities are going to have that `Component` (more than 10%, for example).
```
Comment (magnonellie) [inline book/src/concepts/component.md]: I would consider replacing `have` with `carry`.

Comment (torkleyy) [reply book/src/concepts/component.md]: Done.

**Comment by torkleyy** [comment]
I think that's better now.

**Pull request merged**